src/Chapter2.hs

```diff
@@ -731,7 +740,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: (Eq a, Num a) => a -> [[a]] -> [[a]]
+contains n = filter (\xs -> n `elem` xs)
```

Comment:
You can go one step further and improve the implementation in here even more by passing a partially applied `elem` function. Note, that if you rewrite your lambda in a form `(\xs -> elem n xs)` you can notice an opportunity for eta-reduction.